### DISCUSSION TITLE
Online: chat function, increase lobby name, more players?

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#254] [Suggestions (Multiplayer)]
### Describe the feature you'd like to see added

1. Chat function?
2. Increasing the character limit of the lobby name?
3. While trying to get Radmin to work one time, it loaded up 10 ghost players somehow. Would more than 6 people be possible for online lobbies?
![OR2006C2C_bBE9ySClHX](https://github.com/user-attachments/assets/8a482e54-4c7a-4a86-8811-31b91b4a0c8d)

**Comment by emoose** [comment]
0x65A7B1 (byte) seems to store slot count, changing to 10 shows it as limit but no idea if game would work with it.
(might need to go into Game Settings menu and make a change for it to stick, that'll set it back to 6 though unless you set `4913D9` -> 90 90 90)

The lobby name buffer only has space for 16 chars, but the way it's transmitted to server/clients would allow it to be any size really, maybe can find some way to extend the buffer size.

Chat would be nice, think I did see some stuff for it in the DW code but guess game doesn't make use of it, maybe could just add something so that lobby host also hosts a chat server too which clients could connect to

**Comment by Tenome** [comment]
I never ended up trying the slot count edit, but do you think it would work with only the host making the address edit?
